build/server/konnectors/digiposte.js

```diff
@@ -0,0 +1,243 @@
+'use strict';
```

Comment:
Con you not push a built version of the K ? instead the rules have changed : - )